_security-plugin/multi-tenancy/mt-agg-view.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Multi-tenancy aggregate view for saved objects
```

Comment:
Should this be changed to aggregation view to be consistent? The feature flag is `opensearch_security.multitenancy.enable_aggregation_view `

Reply:
I agree that we need to make the name of the feature consistent. "Aggregate view" is probably the right term. However, I see both "aggregate view for" and "aggregate view of" in the following text.